src/modules/returns/lib/jobs/xml-to-json.js

```diff
@@ -40,9 +41,12 @@ const handleReturnsXmlToJsonStart = async job => {
   const evt = await event.load(job.data.eventId);
 
   try {
-    const s3Object = await returnsUpload.getReturnsS3Object(job.data.eventId);
+    const [s3Object, userResponse] = await Promise.all([
+      returnsUpload.getReturnsS3Object(job.data.eventId),
+      idmConnector.usersClient.getUserByUserName(evt.issuer)
+    ]);
 
-    const json = mapXml(parseXmlFile(s3Object.Body));
+    const json = await xmlToJsonMapping.mapXml(parseXmlFile(s3Object.Body), userResponse.data[0]);
```

Comment:
Do we need to check the user has been loaded correctly before passing to the mapping function?